### ISSUE TITLE
Enhancement request: skipping parts of paths

### ISSUE THREAD

**Issue by callegar** [vitalif/grive2#173]
Suppose that in my Google drive I have some `dir3` inside some `dir2` inside some `dir1` and That I only want to sync `dir3`

if I do

```
cd foo
grive -s dir1/dir2/dir3
```

then in `foo` I get `dir1/dir2/dir3`

would it be possible to have an option so that the path is squashed and I only get `dir3` in `foo`?

I am thinking of something in the lines of `--squash-path-levels 2 -s dir1/dir2/dir3` to get rid of 2 directory levels or `-from dir1/dir2 -s dir3`

**Comment by vitalif** [comment]
Yeah, interesting idea. Everything is possible, it just needs to be coded :)

**Comment by vitalif** [comment]
Another way of doing this may be `grive --source dri1/dir2/dir3 --destination <local_folder>`
